packages/beacon-node/src/chain/stateCache/lruBlockStateCache.ts

```diff
@@ -0,0 +1,174 @@
+import {toHexString} from "@chainsafe/ssz";
```

Comment:
this is very nice, but I think the name LRU is misleading.
IMO this is a FIFO cache with special handling for the head.
I'd rename instances of LRU (and lru) to FIFO (and fifo).